### PR TITLE
Initial commit of docker

### PR DESCRIPTION
**Pull request by hayzer**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM alpine
+MAINTAINER tmaier@autheos.com
+
+ARG JSPATH
+ARG EAN
+ENV JSPATH ${JSPATH}
+ENV EAN ${EAN}
```
Comment (hayzer) [inline docker/Dockerfile]: You have to add LANGUAGE here, both for ARG and ENV. Else in case you will use them, `builder.py` want see it.

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+ARG JSPATH
+ARG EAN
+ARG LANGUAGE
+ENV JSPATH ${JSPATH}
+ENV EAN ${EAN}
+ENV LANGUAGE ${LANGUAGE}
+RUN apk update && \
+    apk add python py-pip nodejs && \
+    pip install tempita && \
```
Comment (hayzer) [inline docker/Dockerfile]: Why did you remove the strict-ssl option?

Comment (Deltanic) [inline docker/Dockerfile]: I did not, this was added after merging. It's in master: https://github.com/autheos/embedcode/blob/master/docker/Dockerfile#L16
